spec/locations/api_patch_locations_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../spec_helper'
+require 'securerandom'
+require 'pry'
+
+RSpec.describe 'PATCH/locations' do
+  let(:api_client) { ApiClient.new }
+  let(:random_location) { JSON.parse(api_client.locations.body).sample }
+
+
+  context 'when valid request' do
+    it 'verifies updated location with new name' do
+      name_for_update = random_location['name']
+      new_locations_name = SecureRandom.hex(10)
+      # binding.pry
+      patch_response = api_client.locations(type_request: :patch, parameter: name_for_update, body_opts: { "new_name": new_locations_name })
+      response = api_client.locations(parameter: new_locations_name) # get with old name status 400
```

Comment:
Can be renamed from response to get_response_new_name
Or just used in expect 
expect(api_client.locations(parameter: new_locations_name).status).to eq(200)
And the same for 
expect(api_client.locations(parameter: name_for_update).status).to eq(200)